pkg/odo/cli/delete/component.go

```diff
@@ -42,6 +63,28 @@ func (o *ComponentOptions) Run() error {
 
 // deleteNamedComponent deletes a component given its name
 func (o *ComponentOptions) deleteNamedComponent() error {
+	log.Info("Searching resources to delete, please wait...")
+	list, err := o.clientset.DeleteClient.ListResourcesToDelete(o.name, o.clientset.KubernetesClient.GetCurrentNamespace())
+	if err != nil {
+		return err
+	}
+	if len(list) == 0 {
+		log.Infof("No resource found for component %s\n", o.name)
+		return nil
+	}
+	log.Info("The following resources will be deleted: ")
+	for _, resource := range list {
+		fmt.Printf("\t%s: %s\n", resource.GetKind(), resource.GetName())
+	}
+	if o.forceFlag || ui.Proceed("Are you sure you want to delete these resources?") {
+		failed := o.clientset.DeleteClient.DeleteResources(list)
+		for _, fail := range failed {
+			log.Warningf("Failed to delete the %q resource: %s\n", fail.GetKind(), fail.GetName())
+		}
+		return nil
```

Comment:
Can you print a success message here saying that the resources were deleted successfully? Perhaps add a spinner?